spec/spec_helper.rb

```diff
@@ -1,4 +1,7 @@
 require 'support/login_helper'
+require 'coveralls'
+
+Coveralls.wear!('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.